tests/unit/core/data/test_module.py

```diff
@@ -58,3 +62,34 @@ def test_init(
         OTXDataModule(task=task, config=fxt_config)
 
         assert mock_otx_dataset_factory.create.call_count == 3
+
+    @pytest.fixture()
+    def fxt_real_tv_cls_config(self) -> DictConfig:
```

Comment:
Thanks for update!
I'm concerned that it will become an invalid test case later if some parts in `torcivision_cls.yaml` that have caused this issue are updated.
I wonder if there is a slight possibility that this could happen.

Reply:
Honestly, my intention was the exact opposite of your comment. I'm hoping this test will validate the scenario where `torcivision_cls.yaml` receives updates, potentially incorporating custom resolvers in the future. If any issues arise, this test will serve a validation role.